examples/web/nextjs/pages/mongodb-data-access.js

```diff
@@ -0,0 +1,37 @@
+// :snippet-start: mongo-data-access
+import { useEffect, useState } from "react";
+import * as Realm from "realm-web";
+
+function MongoDbDataAccess({ name }) {
+  const [plant, setPlant] = useState();
+  const app = Realm.App.getApp(process.env.NEXT_PUBLIC_APP_ID);
+
+  useEffect(() => {
+    if (app?.currentUser) {
```

Comment:
The `app` const shouldn't ever be nullish.

```suggestion
    if (app.currentUser) {
```

Reply:
in the shared `getApp` hook, it'll returns `null` for the initial server-side render

Reply:
Is there a way to work around that.